config-templates/redhat/local-storage/1.0.0/custom-parameters.json

```
@@ -0,0 +1,121 @@
+    },
+    {
+      "description": "The 'value' of the worker node for file 'key=value' label.",
+      "displayname": "File Nodes Label Key Value",
+      "name": "file-nodes-label-value",
+      "required": "false",
+      "category": "config",
+      "type": "csv"
+    },
+    {
+      "description": "The local storage device path for local storage file. Example: '/dev/sdc'. Required when 'auto-discover-devices-file' is set to 'false'.",
+      "displayname": "Device Path for Local Storage File",
+      "name": "file-devicepath",
+      "place-holder": "/dev/sdc,/dev/sde",
+      "required": "false",
+      "category": "config",
+      "type": "csv"
+    },
+    {
+      "description": "The file system type. Specify 'ext3', 'ext4', or 'xfs'.",
+      "displayname": "File System type",
+      "name": "fstype",
+      "default": "ext4",
+      "regex": "^(?:ext(?:3|4)|xfs)$",
+      "min-length": 3,
+      "max-length": 4,
+      "supported-values": ["ext3", "ext4", "xfs"],
+      "required": "true",
+      "category": "config"
+    },
+    {
+      "description": "Set to 'true' to install local storage block driver.",
+      "displayname": "Install Local Storage Block driver",
+      "name": "install-local-storage-block",
+      "default": "true",
+      "type": "boolean",
+      "regex": "^(?:true|false)$",
+      "min-length": 4,
+      "max-length": 5,
+      "supported-values": ["true", "false"],
+      "required": "true",
+      "category": "config"
+    },
+    {
+      "description": "Set to 'true' if you want to automatically discover and use the storage volumes on your worker nodes for block storage.",
+      "displayname": "Automatic storage volume discovery for local storage block",
+      "name": "auto-discover-devices-block",
+      "default": "false",
+      "type": "boolean",
+      "regex": "^(?:true|false)$",
+      "min-length": 4,
+      "max-length": 5,
+      "supported-values": ["true", "false"],
+      "required": "true",
+      "category": "config"
+    },
+    {
+      "description": "The 'key' of the worker nodes for block 'key=value' label.",
+      "displayname": "Block Nodes Label Key",
+      "name": "block-nodes-label-key",
+      "required": "false",
+      "category": "config"
+    },
+    {
+      "description": "The 'value' of the worker nodes for block 'key=value' label.",
+      "displayname": "Block Nodes Label Key Values",
+      "name": "block-nodes-label-value",
+      "required": "false",
+      "category": "config",
+      "type": "csv"
+    },
+    {
+      "description": "The local storage device path for local storage block. Example: '/dev/sdc'. Required when 'auto-discover-devices-block' is set to false.",
+      "displayname": "Device Path for Local Storage Block",
+      "name": "block-devicepath",
+      "place-holder": "/dev/sdc,/dev/sde",
+      "required": "false",
+      "category": "config",
+      "type": "csv"
+    },
+    {
+      "description": "Local volume name.",
```

```suggestion
      "description": "Local volume name.",
```
this can be any name right? could we just populate this for them?

---

removed this param in next PR.